mod.spec.js

```diff
@@ -1,10 +1,9 @@
 const assert = require('chai').assert;
-const KeyCodeLib = require('../index');
-const KeyCodeDist = require('../dist/keycode.min.js');
```

Comment:
We'd still need to test the minified code here so why should it be removed?

Reply:
The best way to test the minified build is using something like karma which tests it in a real browser.

Reply:
OK, got it.